### PR TITLE
Few more typos

### PR DESCRIPTION
**Pull request by richdouglasevans**
Just fixing some wee typos I spotted while reading the excellent documentation.

Good work Team Cypress! 👍

### CODE REVIEW

File: source/api/utilities/blob.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ cy.get("input[type=file]").then(($input) => {
 
 ## Getting dataUrl string
 
-**create an `img` element and set its `src` to the `dataUrl`**
+**Create an `img` element and set its `src` to the `dataUrl`**
```
Comment (richdouglasevans) [inline source/api/utilities/blob.md]: Not a typo: just capitalizing the first word in the sentence for consistency with other examples in the docs.

File: source/api/utilities/moment.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ cy.moment()  // Errors, cannot be chained off 'cy'
 
 # Examples
 
-**test that the span contains formatted text for today**
+**Test that the span contains formatted text for today**
```
Comment (richdouglasevans) [inline source/api/utilities/moment.md]: Likewise, consistent capitalization.

**Comment by jennifer-shehane** [comment]
Hi @richdouglasevans, it looks like you may have deleted the feature branch/fork of the work you did before I was able to merge. This seems to have broken our checks (you know we love automated tests!). 

Here is a workaround to retrieve your deleted fork. You will unfortunately have to make a new PR. https://github.com/isaacs/github/issues/168#issuecomment-374201226

**Comment by richdouglasevans** [comment]
Well darn, sorry about that Jennifier. I'll sort it.

**Pull request closed without merging**